src/core/Select/Select.tsx

```diff
@@ -273,36 +276,38 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
   }, [options, value]);
 
   return (
-    <div
-      className={cx('iui-select', { [`iui-${size}`]: !!size }, className)}
-      aria-expanded={isOpen}
-      aria-haspopup='listbox'
-      style={style}
-      {...rest}
+    <DropdownMenu
```

Comment:
You will need to move this DropdownMenu inside the wrapper if you want the `focus-within` logic to work. Right now there is no focus ring when the menu is open.

Something like this:
![image](https://user-images.githubusercontent.com/9084735/152988841-af72259a-dd2f-43fd-9bac-05176253bc9b.png)

Reply:
changed structure

Reply:
This works for now, but you should start thinking about how to tackle this once we add `appendTo={document.body}` to Popover (#514)

Reply:
> This works for now, but you should start thinking about how to tackle this once we add `appendTo={document.body}` to Popover (#514)

Should we add some class to make it look like focused?